tests/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc",
     "lint": "eslint src --max-warnings 0",
-    "playwright": "npx playwright test",
+    "playwright": "npx playwright test --ui",
```

Comment:
Probably not intentional

Reply:
It is actually! Since we really only use this command to check why an individual test is failing, I figured we may as well add the `--ui` flag to make this easier. I would previously set `--headed` instead, but `--ui` is much nicer.

Reply:
Alright, well then the playwright:ui task is redudant then